toolkit/scripts/toolchain/build_official_toolchain_rpms.sh

```diff
@@ -32,6 +32,7 @@ start_record_timestamp "prep_files"
 MARINER_LOGS=$MARINER_BUILD_DIR/logs
 TOOLCHAIN_LOGS=$MARINER_LOGS/toolchain
 TOOLCHAIN_BUILD_LIST=$TOOLCHAIN_LOGS/build_list.txt
+TOOLCHAIN_BUILT_RPMS_LIST=$TOOLCHAIN_LOGS/build_rpms_list.txt
```

Comment:
Do we clean this directory at all? I can't find it anywhere. We might end up with multiple runs appended together here if not.

Reply:
Good point. I'll update that once your PR is in.